hs_core/templates/pages/genericresource.html

```diff
@@ -23,6 +23,7 @@
         <input type="hidden" id="keywords-string" value="{% for k, v in subjects_form.initial.iteritems  %}{{ v }}{% endfor %}">
         <input type="hidden" id="short-id" value="{{ page.get_content_model.short_id }}">
         <input type="hidden" id="static-url" value="{{ STATIC_URL }}">
+        <input type="hidden" id="resource-url" value="https://helium.commonssshare.org{{ cm.get_absolute_url }}">
```

Comment:
This input hidden field can be removed since you are not using it on insignia js code.